detox/test/package.json

```diff
@@ -26,7 +26,6 @@
     "mocha": "^5.0.0"
```

Comment:
Let's update to 6 if 5 has issues.

Reply:
Looks like a glitch. Check master, if we already have `^6.1.3` there in `detox/test/package.json`.
Anyways, thanks, I saw some of the example projects had `4.0.0`, so your comment is relevant!

Reply:
Resolved in #1308 and #1298.